### PR TITLE
Adding base styles for article header

### PR DESCRIPTION
**Pull request by tpitre**

### CODE REVIEW

File: source/scss/_library/_module.header.scss [hunk 1]
```diff
@@ -115,6 +118,19 @@
...
+
+  // &::-webkit-scrollbar {
+  //   width: 8px;
+  // }
+
+  // &::-webkit-scrollbar-thumb {
+  //   border-radius: 6px;
+  //   background-color: $c-gray;
+  // }
+
```
Comment (noslouch) [inline source/scss/_library/_module.header.scss]: what's going on here? custom scrollbar?

**Comment by tpitre** [comment]
@noslouch Agree on scrollbar. That's why I stopped pushing it further but kept the code in just in case it became an issue. 

Re: social - I definitely think they should be the same molecule. We can add conditionals to the classes to hook specific styles into their respective versions as well as conditionally apply labels based on where it's being displayed. 

I think we already have an `article-header-meta`? When I think "meta" I think things like byline, pub date, categories, tags, etc... Social could definitely be a part of that, but if we wanted to get more generic, I'd recommend something like "tools" or "utilities" since they're actionable.

**Comment by noslouch** [comment]
Ah, sorry I'm talking about the bits that go in the global header here: https://app.zeplin.io/project/5a9dc653aaaa927d5696de7c/styleguide/components?cseid=5c785f06a3cb9bbc3c9c5672

The naming is definitely a bit confusing, but now I've renamed them in branch so the molecules we have for the header are:
- `meta/header-meta`
- `meta/header-article-meta`
- `meat/header-overlay-gallery-meta`

**Comment by tpitre** [comment]
@noslouch Yup, got it. Refactoring now.

**Comment by tpitre** [comment]
@noslouch You think this one's good to go now?

**Pull request merged**